### ISSUE TITLE
Better handling of no connection table on start

### ISSUE THREAD

**Issue by philipstarkey** [labscript-suite-oldfinal1/blacs#30]
**Original report ([archived issue](https://labscript-suite.github.io/labscript-suite-bitbucket-archive/#!/labscript_suite/blacs/issues/30)) by Jan Werkmann (Bitbucket: [PhyNerd](https://bitbucket.org/%7B44c4905c-2b90-4045-a5f1-652b8e228626%7D/), GitHub: [PhyNerd](https://github.com/PhyNerd)).**

----------------------------------------
Currently one is greeted with a big exception when starting BLACS without a connection table. It would be much nicer to have the connection table compilation window show up instead with a helpful message why BLACS didn't start properly.
Users that are new to labscript would definitely profit from this.

**Comment by philipstarkey** [comment]
**Original comment by Philip Starkey (Bitbucket: [pstarkey](https://bitbucket.org/%7B48af65db-e5fc-459c-a7eb-52eb1f9a5690%7D/), GitHub: [philipstarkey](https://github.com/philipstarkey)).**

----------------------------------------

* changed state from "new" to "duplicate"

Duplicate of #8.

**Comment by philipstarkey** [comment]
**Original comment by Philip Starkey (Bitbucket: [pstarkey](https://bitbucket.org/%7B48af65db-e5fc-459c-a7eb-52eb1f9a5690%7D/), GitHub: [philipstarkey](https://github.com/philipstarkey)).**

----------------------------------------
It may not be obvious from the title, but the proposed solution to issue #8 is to make BLACS launchable without a connection table (either broken or missing) so that you can access the connection table compilation options (and maybe include some helpful error messages!)

I think this is possibly blocked by improving the plugin interface (implementing something that allows you to enable/disable plugins)
